### ISSUE TITLE
v4.0.0 - Portal EmailSettings - Hostname not resolved

### ISSUE THREAD

**Issue by lorenzmeyer77** [Esri/arcgis-powershell-dsc#413]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Module Version

* 4.0.0

### Affected Resource(s)

* ArcGISPortalSettings

### Configuration Files

Just add an emailsettings property:

            "EmailSettings": {
                "SMTPServerAddress": "xxxxxx",
                "From": "no-reply@google.com",
                "Label": "Portal TEST",
                "AuthenticationRequired": false,
                "EncryptionMethod": "SSL"
            }

### Expected Behavior
The EmailSettings are set in portal like it worked with the DSC Module 3.3.2.

### Actual Behavior
DSC Error in ArcGISPortalSettings --> the hostname ist missing!
```
09.09.2022 09:36:46: Cannot bind parameter 'Uri'. Cannot convert value "https://:7443/arcgis/portaladmin/system/emailSettings/update" to type "System.Uri". Error: "Invalid URI: The hostname could not be parsed."
09.09.2022 09:36:46: PowerShell DSC resource ArcGIS_PortalSettings  failed to execute Set-TargetResource functionality with error message: Request to https://:7443/arcgis/portaladmin/system/emailSettings/update failed. Response returned NULL 
09.09.2022 09:36:46: The SendConfigurationApply function did not succeed.

```

### Steps to Reproduce

Just add some email settings for portal an run them with DSC 4.0.0
### Important Factoids

Running in Azure (westeurope)

**Comment by lorenzmeyer77** [comment]
@scma-esrich fyi

**Comment by lorenzmeyer77** [comment]
Without EMailSettings it works in DSC 4.0.0

**Comment by cameronkroeker** [comment]
Hi @lorenzmeyer77,

Thanks for the submission. This will be addressed in the next release, in the meantime, the following workaround can be used to allow the email settings to be applied successfully.

**Update:**

https://github.com/Esri/arcgis-powershell-dsc/blob/4a33aabf965e2098699bddc499ce1d05003e992d/Modules/ArcGIS/DSCResources/ArcGIS_PortalSettings/ArcGIS_PortalSettings.psm1#L340

**To:**
```powershell
Update-PortalEmailSettings -PortalHostName $PortalFQDN -SMTPServerAddress $EmailSettingsSMTPServerAddress -From $EmailSettingsFrom -Label $EmailSettingsLabel -AuthenticationRequired $EmailSettingsAuthenticationRequired -Credential $EmailSettingsCredential -SMTPPort $EmailSettingsSMTPPort -EncryptionMethod $EmailSettingsEncryptionMethod -Token $token.token -Referer $Referer -Verbose
```

**Comment by cameronkroeker** [comment]
Hello @lorenzmeyer77 and @scma-esrich,

This issue has been addressed in [v4.0.1](https://github.com/Esri/arcgis-powershell-dsc/releases/tag/v4.0.1).

Thanks,
Cameron K.
